transformer_engine/pytorch/csrc/extensions.cu

```diff
@@ -12,43 +12,20 @@
 constexpr int block_size = 512;
 constexpr int ctas_per_sm = 4;
 
-// convert QKV layout to enum
-NVTE_QKV_Layout get_nvte_qkv_layout(const std::string qkv_layout) {
-  if (qkv_layout == "not_interleaved") {
-      return NVTE_QKV_Layout::NVTE_NOT_INTERLEAVED;
-  } else if (qkv_layout == "qkv_interleaved") {
-      return NVTE_QKV_Layout::NVTE_QKV_INTERLEAVED;
-  } else if (qkv_layout == "kv_interleaved") {
-      return NVTE_QKV_Layout::NVTE_KV_INTERLEAVED;
-  } else {
-      NVTE_ERROR("Invalid QKV layout. \n");
-  }
-}
-
-// convert bias type to enum
-NVTE_Bias_Type get_nvte_bias_type(const std::string bias_type) {
-  if (bias_type == "no_bias") {
-      return NVTE_Bias_Type::NVTE_NO_BIAS;
-  } else if (bias_type == "pre_scale_bias") {
-      return NVTE_Bias_Type::NVTE_PRE_SCALE_BIAS;
-  } else if (bias_type == "post_scale_bias") {
-      return NVTE_Bias_Type::NVTE_POST_SCALE_BIAS;
-  } else {
-      NVTE_ERROR("Invalid bias type. \n");
-  }
-}
-
-// convert attn mask type to enum
-NVTE_Mask_Type get_nvte_mask_type(const std::string mask_type) {
-  if (mask_type == "padding") {
-      return NVTE_Mask_Type::NVTE_PADDING_MASK;
-  } else if (mask_type == "causal") {
-      return NVTE_Mask_Type::NVTE_CAUSAL_MASK;
-  } else if (mask_type == "no_mask") {
-      return NVTE_Mask_Type::NVTE_NO_MASK;
-  } else {
-      NVTE_ERROR("Invalid attention mask type. \n");
-  }
+// check if there is a fused attention backend available
+NVTE_Fused_Attn_Backend is_fused_attn_available(
```

Comment:
The function name is somewhat misleading:
```suggestion
NVTE_Fused_Attn_Backend get_fused_attn_backend(
```

Reply:
Done, renamed this and the nvte function.